frontend/src/Components/PageLayout/HeaderFiles/HeaderPage/Header.js

```diff
@@ -0,0 +1,89 @@
+import { Box, Typography } from "@mui/material";
+import React from "react";
+import { Actions } from "../../../../Utils/Actions";
+import { useLocation, useNavigate } from "react-router-dom";
+import { useSearchFilter } from "../HeaderTabs/SearchFilterProvider";
+import DrawerMenu from "./DrawerMenu";
+import NavigationTabs from "./NavigationTabs";
+import SearchBox from "./SearchBox";
+import HeaderActions from "./HeaderActions";
+
+export default function Header() {
+  const { searchTerm, setSearchTerm, toggleFilter } = useSearchFilter();
+  const navigate = useNavigate();
+  const location = useLocation();
+  const {
+    open,
+    user,
+    totalQuantity,
+    handleSignOut,
+    handleChange,
+    toggleDrawer,
+  } = Actions();
+
+  const showSearch = ["/", "/products", "/products/:id"].some((path) =>
+    location.pathname.includes(path)
+  );
+  const showFilter = location.pathname === "/products";
+  const showBackButton = location.pathname !== "/";
+  const showDrawer = location.pathname === "/";
```

Comment:
Why this? can we make something like
```suggestion
  const showDrawer = true;
```

Reply:
The drawer is only shown in Home Page , in rest of the Pages it will be replaced by Back Arrow button.